src/main/java/org/folio/innreach/mapper/ItemContributionOptionsConfigurationMapper.java

```diff
@@ -0,0 +1,12 @@
+package org.folio.innreach.mapper;
+
+import org.folio.innreach.domain.entity.ItemContributionOptionsConfiguration;
+import org.folio.innreach.dto.ItemContributionOptionsConfigurationDTO;
+import org.mapstruct.Mapper;
+
+@Mapper(componentModel = "spring")
+public interface ItemContributionOptionsConfigurationMapper {
+  ItemContributionOptionsConfiguration toEntity(ItemContributionOptionsConfigurationDTO itemContributionOptionsConfigurationDTO);
+
+  ItemContributionOptionsConfigurationDTO toDto(ItemContributionOptionsConfiguration createdItemContributionOptionsConfiguration);
```

Comment:
There is a difference in field naming of metadata object in entity and DTO. So the mapping between those fields has to be manually defined. See `LibraryMappingMapper` for example